interview-questions.md

```diff
@@ -9,47 +9,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
 Your answer:
+JSX is a javascript markdown syntax extension that works similar like HTML. JSX is a building block of REACT components and what allows React to create user interfaces. It looks like a combination of JavaScript and an extended set of HTML.
 
 Researched answer:
+JSX is a React extension to Javascript. It allows JavaScript and HTML to co-exist. This allows component rendering using syntax similar to HTML.
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-Your answer:
+Your answer: 
+A react app is made of thousands of code snippets called node modules, that has to be organized in a certain way. So in order to create a new React app we have to download a package containing these modules. Yarn (yet another resource negotiator) is the command that will download these modules and create a file structure for a React app. 
 
 Researched answer:
+YARN is a package that contains node modules , it allows the use and share of code with developers around the world. The code is shared through a package and includes j.son file which describes the package. 
 
 3. What is an anonymous function in JavaScript?
 
 Your answer:
+Anonymous function is a function that does not have a name. 
 
 Researched answer:
+An anonymous function is a function that is not bound to an identifier. For example function() is a function that is missing its name, an anonymous function. 
 
 4. What is the difference between a class and an object?
 
 Your answer:
+An object is a data type that contains data like names, keys values etc. A class is a template from which objects are created, it is a holder for what the object is going to look like. Similar to a blueprint and a house, house being the object. 
 
 Researched answer:
+A class is a blueprint, it defines the different properties that an object can have, but it doesn't give it any data or information. An object is a specific version of a class that has its information filled.
```

Comment:
Yea, an Object is an instance of a Class. be sure to utilize the vocabulary you are learning. Getting a job will depend on it!